src/libs/config.js

```diff
@@ -1,10 +1,10 @@
 import _ from 'lodash/fp'
 import * as Utils from 'src/libs/utils'
 
-
-const loadConfig = _.memoize(async () => {
+let loadedConfig
+export const loadConfig = _.memoize(async () => {
```

Comment:
Memoize still a benefit?

Reply:
Nope, not necessary any more since we only call it once. Good catch.